src/components/CromwellModal.js

```diff
@@ -24,18 +21,19 @@ const titleId = 'cromwell-modal-title'
 // (or find another way to reduce code duplication).
 export const CromwellModalBase = Utils.withDisplayName('CromwellModal')(
   ({
-    onDismiss, onSuccess, apps, workspace, workspace: { workspace: { namespace, bucketName, name: workspaceName, googleProject } },
+    onDismiss, onSuccess, apps, appDataDisks, workspace, workspace: { workspace: { namespace, bucketName, name: workspaceName, googleProject } },
     isAnalysisMode = false
   }) => {
     const app = getCurrentAppForType(tools.cromwell.appType)(apps)
     const [loading, setLoading] = useState(false)
+    const currentDataDisk = currentPersistentDisk(tools.cromwell.appType, apps, appDataDisks)
 
     const createCromwell = _.flow(
       Utils.withBusyState(setLoading),
       withErrorReportingInModal('Error creating Cromwell', onDismiss)
     )(async () => {
       await Ajax().Apps.app(googleProject, Utils.generateAppName()).create({
-        defaultKubernetesRuntimeConfig, diskName: Utils.generatePersistentDiskName(), diskSize: defaultDataDiskSize,
+        defaultKubernetesRuntimeConfig, diskName: currentDataDisk ? currentDataDisk.name : Utils.generatePersistentDiskName(), diskSize: defaultDataDiskSize,
```

Comment:
What would you think about:
```suggestion
        defaultKubernetesRuntimeConfig, diskName: currentDataDisk?.name ?? Utils.generatePersistentDiskName(), diskSize: defaultDataDiskSize,
```
(`??` is [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator))

Reply:
Huh, that looks like magic to me. :)

Reply:
Changed both locations.

Reply:
I guess I do wonder though if this goes against our policy of trying to make the code familiar/friendly to folks who aren't JavaScript experts. Tertiary operators are a common pattern in many different languages, but this nullish coalescing is likely to be unfamiliar to pretty much everyone.

Reply:
I think your question is valid @cahrens and we should at least clarify what dev audience exactly we optimize for.

That being said, I had been wanting `??` all along not knowing it's existed 🤩 , for which, thank you @zarsky-broad!

Reply:
Would you consider my suggestion with `||` instead of `??`, then?

Reply:
@zarsky-broad I don't understand what that would look like. But I already made the change you proposed with ??, and I don't feel like it's worth backing it out. But I do think it's worth discussing who our dev audience is, as there is tension between using the newer JavaScript language constructs and making the code easily understandable to people who aren't JS experts.

Reply:
I ended up going back the ternary structure as we discussed in the mob.